FBSimulatorControl/Management/FBSimulatorBridge.m

```diff
@@ -55,7 +55,6 @@ + (nullable instancetype)bridgeForSimulator:(FBSimulator *)simulator error:(NSEr
   }
 
   // Load Accessibility, return early if this fails
-  [bridge enableAccessibility];
```

Comment:
Do we **really** need this line to be deleted? In my understanding, if we decided to remove some method call due to its deprecation - we have to implement its replacement that will do the same things as the original method

Reply:
Fixed this. Added checking that bridge has this selector before invoking it.